test/integration/ReplicationTest.cc

```diff
@@ -58,53 +58,78 @@ namespace {
         // Put it into rdfs.
         sleep(10);
         system("hdfs dfs -fs hdfs://localhost:5351 -D dfs.blocksize=1048576 -copyFromLocal expected_testfile1234 /g");
-        // Read it from rdfs.
+//        // Read it from rdfs.
         sleep(10);
         system("hdfs dfs -fs hdfs://localhost:5351 -cat /g > actual_testfile1234");
-        // Check that its contents match.
+//        // Check that its contents match.
         sleep(10);
         ASSERT_EQ(0, system("diff expected_testfile1234 actual_testfile1234 > /dev/null"));
 
         // Start a new server
-        std::string dnCliArgs = std::to_string(xferPort + 4) + " " + std::to_string(ipcPort + 4) + " tfs" + std::to_string(4) + " &";
-        std::string cmdLine = "bash -c \"exec -a ReplicationTestServer" + std::to_string(4) + " /home/vagrant/rdfs/build/rice-datanode/datanode " +
-                              dnCliArgs + "\" & ";
-        system("truncate tfs4 -s 1000000000");
+//        std::string dnCliArgs = std::to_string(xferPort + 4) + " " + std::to_string(ipcPort + 4) + " tfs" + std::to_string(4) + " &";
```

Comment:
If this code isn't used anymore, please delete the lines instead of commenting it out